demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -226,10 +248,12 @@ def _is_pack_meta_file_structure_valid(self):
                     return False
 
             # check validity of pack metadata mandatory fields
-            name_field = metadata.get(PACK_METADATA_NAME, '').lower()
-            if not name_field or 'fill mandatory field' in name_field:
```

Comment:
Why did you remove the second condition?

Reply:
Sorry. it's was my mistake, I thought it was unnecessary